tests/unit/core.spec.js

```diff
@@ -2,38 +2,96 @@
 Stub test file
 TODO: Write unit tests for all interfaces
 */
-import { JSONEditor } from '../../src/core';
-
-const schema = 
-{
-  type:'object',
-  properties:
-  {
-    name:{type:"string"}
+import { JSONEditor } from '../../src/core'
+
+const schema = {
+  type: 'object',
+  properties: {
+    name: { type: 'string' }
   }
 }
 
-
-describe("JSONEditor", function() {
+describe('JSONEditor', function () {
+  let element
+  let editor
 
   // inject the HTML fixture for the tests
-  beforeEach(function() {
-    var fixture = '<div id="fixture"></div>';
+  beforeEach(() => {
+    const fixture = '<div id="fixture"></div>'
 
-    document.body.insertAdjacentHTML(
-      'afterbegin', 
-      fixture);
-  });
+    document.body.insertAdjacentHTML('afterbegin', fixture)
+    element = document.getElementById('fixture')
+  })
 
   // remove the html fixture from the DOM
-  afterEach(function() {
-    document.body.removeChild(document.getElementById('fixture'));
-  });  
-
-  it("should create an editor", function(){
-    var element = document.getElementById('fixture');
-    console.log("Attempting to create new JSONEditor");
-    var editor = new JSONEditor(element, {schema:schema });
-    expect(editor).toBeTruthy();
-  });
-})
\ No newline at end of file
+  afterEach(() => {
+    document.body.removeChild(document.getElementById('fixture'))
+  })
+
+  it('should create an editor', () => {
+    console.log('Attempting to create new JSONEditor')
+    editor = new JSONEditor(element, { schema: schema })
+    expect(editor).toBeTruthy()
+  })
+
+  it('can add custom iconlib', () => {
+    const customeMapping = {

```

Comment:
could you change this to `customMapping`

Reply:
Thanks for pointing out. fixed.